stix2/registration.py

```diff
@@ -1,11 +1,72 @@
-import itertools
 import re
 
 from . import registry, version
 from .base import _DomainObject
 from .exceptions import DuplicateRegistrationError
-from .properties import _validate_type
-from .utils import PREFIX_21_REGEX, get_class_hierarchy_names
+from .properties import (
+    ListProperty, ObjectReferenceProperty, ReferenceProperty, _validate_type,
+)
+from .utils import PREFIX_21_REGEX
+
+
+def _validate_ref_props(props_map, version, is_observable20=False):
```

Comment:
`version` here is no longer used.  I think this param can be deleted.

Reply:
Ah yes, good catch

Reply:
Done in the new diff

Reply:
Done in the new diff